### ISSUE TITLE
Cannot activate 2FA authentication

### ISSUE THREAD

**Issue by sorinsfirlogea** [phpmyadmin/phpmyadmin#16462]
When trying to activate the 2FA authentication I constantly receive this 500 error:
Error in processing request
Error code: 500
Error text: Internal Server Error (rejected)
It seems that the connection to server has been lost. Please check your network connectivity and server status.

In error.log:
Call to undefined method BaconQrCode\\Renderer\\Image\\SvgImageBackEnd::getQRCodeInline() in /var/www/html/phpmyadmin/vendor/pragmarx/google2fa-qrcode/src/Google2FA.php:63

OS: Ubuntu 16.04.1
PHP: 7.4.12
Web server: Apache 2.4
phpMyAdmin: 5.0.4 all_languages
DB: MySQL 5.7.32

**Comment by williamdes** [comment]
Hi @sorinsfirlogea 
According to my project https://github.com/code-lts/phpmyadmin-releases#readme

You did extract your zip/tar in an incoherent way, please delete all phpMyAdmin files except the config file and try once more :)

See: https://github.com/code-lts/phpmyadmin-releases/blob/ff42facc94c1f60f8b1e5e2eb2737226725aa18e/vendor/pragmarx/google2fa-qrcode/src/Google2FA.php#L63

See: https://github.com/code-lts/phpmyadmin-releases/blob/upstream/version/all-languages/5.0.4/vendor/bacon/bacon-qr-code/src/Renderer/Image/SvgImageBackEnd.php

**Comment by sorinsfirlogea** [comment]
Actually, that was not the problem. But you helped me to identify the issue. I got the wrong idea that google2fa-qrcode is not shipped with phpmyadmin and I installed it with composer. Which brings the latest version, not the one you are using. Problem solved. Now I just need to figure why the 2FA sync with Google Authenticator is failing - I remember I had this problem in the past, but I solved it. Can't remember how :-)
